src/fmu/dataio/case.py

```diff
@@ -45,9 +45,6 @@ class CreateCaseMetadata:  # pylint: disable=too-few-public-methods
         description (Optional): Description text as string or list of strings.
     """
 
-    # class variables
-    meta_format: ClassVar[Literal["yaml", "json"]] = "yaml"
```

Comment:
Did not deprecate it here since the `CreateCaseMetdata` should only be used by users through the ERT WF  `WfCreateCaseMetadata`